.gitignore

```diff
@@ -1,5 +1,6 @@
 # Compiled JS
 /built
+/build
```

Comment:
you should delete your /build folder - it's out of date
on Mac you would have been forced to do so, because the BUILD file is the same file - if you submit this change then Mac users will be ignoring BUILD which is wrong